### PR TITLE
Frontend js

### PR DESCRIPTION
**Pull request by nmahabir**
Added the recipe Handlebars to the site, and a template for the Recipe.JS.
Still some functionality to work on.

### CODE REVIEW

File: views/recipe.handlebars [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+
+
+
+
+
+
+</div>
+
+<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
```
Comment (iamdp) [inline views/recipe.handlebars:40]: Import tags should be consolidated to the index file, they are unnecessary on the .handlebars

File: views/recipe.handlebars [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+<script src="/js/recipe.js"></script>
```
Comment (iamdp) [inline views/recipe.handlebars:41]: Not sure if this is necessary, I don't visually see any references to it. Consider removing?

File: public/js/recipe.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+var recipeName = $("#recipeName");
+var recipeImage = $("#recipeImage");
+var prepTime = $("#prepTime");
+var servings = $("#servings");
+
+// The API object contains methods for each kind of request we'll make
+var API = {
+  getRecipe: function(recipe) {
+    return $.ajax({
+      url: "api/recipe",
```
Comment (iamdp) [inline public/js/recipe.js]: Just making a note here that the url will eventually need to be updated to reflect the end point.

**Review by iamdp — APPROVED** [review]
Curious to the intended recipe.js functionality. Good changes, good code. Lets merge.

**Comment by nmahabir** [comment]
Ya that one we should confirm. It was to show the recipr page but i noticed
the recipes are in url form from yummly.

But i think the oroginal purpose was to show the recipe info, so i just
make a click here for recipe so not sure how to proceed with it as yet.

Tonight we can talk some more and I'll make any changes one time.

Thanks for your feedback!
Natalia

**Pull request merged**